src/mess/drivers/hp64k.c

```diff
@@ -349,6 +385,26 @@ void hp64k_state::video_start()
 		m_chargen = memregion("chargen")->base();
 }
 
+// Divisors of K1135 baud rate generator
```

Comment:
I think we already have a device for this type of baud divider, in machine/com8116.h